### PR TITLE
development/yabasic: Updated for version 2.90.5

### PR DESCRIPTION
**Pull request by antonioleal**

### CODE REVIEW

File: development/yabasic/tar.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (willysr) [inline development/yabasic/tar.sh:1]: i don't think this should be included in SBo?

**Review by willysr — COMMENTED** [review]
it's better to have this removed from this PR

File: development/yabasic/slack-desc [hunk 2]
```diff
@@ -16,4 +16,4 @@ yabasic: Yabasic runs under Unix and Windows and has a comprehensive
 yabasic: documentation; it is small, simple, open-source and free.
 yabasic:
 yabasic: Yabasic is very well documented.
-yabasic:
+yabasic: Homepage at http://2484.de/yabasic/
```
Comment (willysr) [inline development/yabasic/slack-desc:19]: please don't put HOMEPAGE in README

**Review by willysr — CHANGES_REQUESTED** [review]
There shouldn't be any URL in the README as well

File: development/yabasic/slack-desc (hunk 2 above)
Comment (antonioleal) [reply development/yabasic/slack-desc:19]: I actually used the URL in the slack-desc as part of my workflow, but ok, I'll arrange something different...

**Comment by antonioleal** [comment]
Ok hope its alright now. I'm upgrading my workflow with new scripts, so some bugs might slip-in...

**Comment by willysr** [comment]
@sbo-bot: build development/yabasic

**Comment by willysr** [comment]
merged

**Pull request closed without merging**